tests/test_pandascore.py

```
@@ -157,12 +157,26 @@ def test_extract_match_data_valid(self, parser):
         result = parser.extract_match_data(match_data, contest_id=1)
         assert result is not None
         assert result["pandascore_id"] == 123456
+        assert result["game"] == "lol"
         assert result["team1"] == "Team A"
         assert result["team2"] == "Team B"
         assert result["team1_id"] == 100
         assert result["team2_id"] == 200
         assert result["best_of"] == 3
 
+    @staticmethod
+    def test_extract_match_data_uses_payload_game_slug(parser):
```

Ruff (0.15.5)

[warning] Missing return type annotation for staticmethod `test_extract_match_data_uses_payload_game_slug`

Add return type annotation: `None`

(ANN205)